package.json

```diff
@@ -441,6 +441,12 @@
           "default": "full",
           "scope": "resource",
           "description": "Response preview output option. 'full' for whole response message(status line, headers and body). 'headers' for response headers(as well as status line). 'body' for response body only. 'exchange' for whole HTTP exchange (request and response)"
+        },
+        "rest-client.showStatusBarItem": {
```

Comment:
minor: how about rename it to showEnvironmentStatusBarItem, since status bar in my extension also includes response duration and size status bars

Reply:
@Huachao Sure, sounds good to me